firmware/Core/Src/telecommands/telecommand_definitions.c

```diff
@@ -249,6 +250,17 @@ const TCMD_TelecommandDefinition_t TCMD_telecommand_definitions[] = {
 
    // ****************** END SECTION: log_telecommand_defs ******************
 
+   // ****************** SECTION: freertos_telecommand_defs ******************
+
+   {
+        .tcmd_name = "retrieve_freertos_metadata",
```

Comment:
Just to keep with the namespace "theme" of "most important info first", would you mind renaming to `freertos_get_metadata`? I'd normally think you name makes more sense, but this way the commands sort a bit nicer. Update function name and string here.

Reply:
Done